aerotech_101smc2/aerotech_101smc2/_aerotech.py

```diff
@@ -48,6 +49,11 @@ def direct_serial_write(self, message):
         self._busy = True
         self._serial_port.write(message.encode())
 
+    def home(self):
+        self._busy = True
+        self._serial_port.write(b"H\n")
+        self._destination= 0.0000
```

Comment:
This works to home the motor, but it doesn't actually implement home according to our specifications.

When homed, hardware should:
1. move to home
2. "zero-out" (I believe this is accomplished by firmware in this case)
3. return to previously instructed destination, counting from new zero position

Your implementation does not return the hardware to an appropriate destination.

Check out the implementation at https://python.yaq.fyi/is-homeable/, it should do everything you need.